CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Security
+- Updated gogo/protobuf to v1.3.2 to address CVE-2021-3121 (https://github.com/cyberark/secrets-provider-for-k8s/pull/285)
```

Comment:
@andytinkham please change this to:
```
- Updated gogo/protobuf to v1.3.2 to address CVE-2021-3121 
  [cyberark/secrets-provider-for-k8s#285](https://github.com/cyberark/secrets-provider-for-k8s/pull/285)
```

to match the changelog style

Reply:
Resolved. Thanks!